paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -920,6 +920,14 @@ internal class PaymentSheetViewModelTest {
         }
     }
 
+    @Test
+    fun `isResourceRepositoryReady emits true on initialize`() = runTest {
+        val viewModel = createViewModel()
+        viewModel.isResourceRepositoryReady.test {
+            assertThat(awaitItem()).isTrue()
+        }
+    }
```

Comment:
We should test this more throughly, e.g. the cases where initialization fails or where it’s slow to initialize, and whether dependent code handles that correctly.